split_expenses_ui/src/services/users.js

```diff
@@ -30,7 +30,7 @@ export const postUsers = (payload) => {
   const url = config["baseUsersURL"];
   const { userName, emailId, phoneNumber } = payload;
   const form = new FormData();
-  form.append('username', userName);
+  form.append('user_name', userName);
```

Comment:
can we use the same naming convention here as well.. we can change the userName in the HTML or wherever we are obtaining it from.

Reply:
Yeah, Updated the same naming convention.